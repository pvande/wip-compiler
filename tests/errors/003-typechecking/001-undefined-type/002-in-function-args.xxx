x := (a : float) => { }
y := (a : triple) => { }
z := (a : int) => { }
