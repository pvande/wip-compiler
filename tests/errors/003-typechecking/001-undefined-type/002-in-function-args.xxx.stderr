

Error: "Definition for type not found"
In [1;37mtests/errors/003-typechecking/001-undefined-type/002-in-function-args.xxx[0m on line [1;37m2[0m

> [0;36my := (a : triple) => { }[0m
  [0;31m          ^^^^^^        [0m
